src/pages/Contents/Contents.js

```diff
@@ -1,11 +1,128 @@
 import React, { Component } from "react";
 import "./Contents.scss";
+import { CaretDownOutlined, PlusOutlined } from "@ant-design/icons";
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { pinkFlag } from "./ActorProfile/data";
 
 class Contents extends Component {
+  constructor() {
+    super();
+    this.state = {
+      addBtnClicked: false,
+      dropdownBtnClicked: false,
+      dropdownBtnColorChanged: false,
+      commentBtnClicked: false
+    };
+  }
   render() {
     return (
       <>
-        <div>sdf</div>
+        <section>
+          <div className="header_box">
+            <div className="dark_box"></div>
+            <div className="main_image"></div>
+          </div>
+
+          <div className="movie_title_section">
+            <img
+              className="movie_title_section_poster"
```

Comment:
이제는 sass에서 nesting이 가능하기 때문에 className을 하나하나 길게 작성할 필요가 없습니다. 
더불어 리엑트에서는 snake-case보다 camelCase 형식의 naming이 주로 이루어집니다. 참고해주세요! 
위 className은 아래와 같이 바꾸어줄 수 잇겠네요! 💁🏻‍♀️ 아래 className들도 모두 수정해주세요!

```suggestion
              className="moviePoster"
```

Reply:
팀끼리 맞춘거라 상의 후 수정하겠습니다~!